pooch/core.py

```diff
@@ -253,6 +255,12 @@ def create(
     ``https://github.com/fatiando/pooch/raw/v0.1/data``). If the version string
     contains ``+XX.XXXXX``, it will be interpreted as a development version.
 
+    Does **not** create the local data storage folder if it doesn't exist. The
```

Comment:
Can simplify a bit:
```suggestion
    Does **not** create the local data storage folder. The
```
Because it also does not create the folder if it does exist :)

Reply:
Good catch :)